src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,168 @@
+$base-color: #000066;
+$white-bg: #fff;
+
+.list-item {
+  padding: 2em 1em;
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 0 2px 5px 5px rgba(0,0,0,0.1);
+  border-radius: 10px;
+  position: relative;
+  background-color: $white-bg;
+  color: #000066;
+}
+
+.profile-details {
+  padding-left: 150px;
+  padding-top: 20px;
+  flex-basis: 50%;
+  max-width: 50%;
+  position: relative;
+  p,
+  h6 {
+    color: #000066;
+  }
+}
+
+.avatar-div {
+  left: 0;
+  position: absolute;
+  top: 0;
+  height: 100%;
+  margin-right: 20px;
+  width: 150px;
+}
+
+.profile-avatar {
+  left: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 120px;
+  height: 120px;
+  object-fit: cover;
+  border-radius: 10px;
+}
+
+.profile-details-heading {
+  color: $base-color;
+  font-weight: 700;
+}
+
+.description {
+  margin-left: 15px;
+}
+
+.profile-contact {
+  flex-basis: 50%;
+  padding-left: 30px;
+}
+
+.contact-mode {
+  color: #000066;
+}
+
+.track-person {
+  position: absolute;
+  right: 20px;
+  top: 0;
+  padding: 5px;
+  
+  p {
+    padding-right: 40px;
+    vertical-align: middle;
+   }
+
+  .fa {
+    position: absolute;
+    top: 0; 
+    color: #000066;
+    font-size: 2.5em;
+    right: 0;
+    background-color: $white-bg;
+ }
+}
+
+.card-links{
+  text-align: left;
+  margin: 1em 0;
+  i {
+    color: $base-color;
+    font-size: 2em;
+    margin: 0 2em 0 0;
```

Comment:
The margin between social icons looks large, so can you please make it 1 rem. `margin: 0 1em 0 0;`